content/Solutions/Integrations/EntraID.md

```diff
@@ -0,0 +1,108 @@
+    There are numerous possible solutions for this purpose.
+    iXsystems does not suggest or support a specific third-party VPN service.
+    For one option, see [Tutorial: Create a site-to-site VPN connection in the Azure portal](https://learn.microsoft.com/en-us/azure/vpn-gateway/tutorial-site-to-site-portal) from Microsoft.
+
+## Configuring TrueNAS
+
+Edit network configuration to enable TrueNAS to reach the Entra ID domain services instance and then join the domain.
+
+1. Configure network connections.
+    Go to **Network** and click **Settings** on the **Global Configuration** widget to open the **Edit Global Configuration** screen.
+
+    {{< trueimage src="/images/SCALE/Credentials/EntraGlobalConfiguration.png" alt="Edit Global Configuration" id="Edit Global Configuration" >}}
+
+    Enter the domain name from Entra ID domain services in **Domain**.
+
+    Enter the VPN public address as a nameserver to route connections to the Entra domain services DNS server.
+
+2. Confirm connectivity.
+    Connect to a local TrueNAS shell or SSH session and enter the {{< cli >}}ping *EntraDomain*{{< /cli >}} command, where *EntraDomain* is the domain name entered above.
+    Verify the ping is successful.
+
+    {{< trueimage src="/images/SCALE/Credentials/EntraPingDomain.png" alt="Confirm Connectivity" id="Confirm Connectivity" >}}
+
+    {{< expand "If the ping fails (click to expand)" "v" >}}
+  1. Go to **Network** and click **Settings** in the **Global Configuration** window.
+  2. Ensure the **DNS Servers** and **Default Gateway** settings match the connection to the Entra domain.
+    {{< /expand >}}
+
+3. Join the domain.
+
+   a. Go to **Credentials > Directory Services** and click **Configure Active Directory** to open the **Active Directory** screen.
+
+   {{< trueimage src="/images/SCALE/Credentials/ActiveDirectoryBasicOptions.png" alt="Active Directory Screen" id="Active Directory Screen" >}}
+
+   b. Enter the Entra domain name in **Domain Name** and the administrator account credentials in **Domain Account Name** and **Domain Account Password**.
+
+   c. Select **Enable** to attempt to join the AD domain immediately after saving the configuration.
+     SCALE populates the **Kerberos Realm** and **Kerberos Principal** fields on the **Advanced Options** settings screen.
+
+   d. Click **Save**.
+
+      TrueNAS can take a few minutes to populate directory information after configuration.
+      To check the join progress, open the <i class="material-icons" aria-hidden="true" title="Assignment">assignment</i> **Task Manager** in the upper-right corner.
+      TrueNAS displays any errors during the join process in the **Task Manager**.
+
+      When the import completes, domain users and groups become available.
+
+4. Verify you are able to lookup domain users.
+
+   From a shell session, enter the command {{< cli >}}wbinfo -u{{< /cli >}}.
```

Comment:
Do not need to refer to the shell. Can use the UI to populate AD users, e.g. Permissions UI for a dataset.